### PR TITLE
added nowIndicatorRender callback firing after nowIndicator has been rendered

### PR DESCRIPTION
**Pull request by zaalbarxx**
added nowIndicatorRender callback firing after nowIndicator has been rendered

### CODE REVIEW

**Comment by zaalbarxx** [comment]
Related to #4663

File: packages/timegrid/src/TimeGrid.ts [hunk 1]
```diff
@@ -547,6 +547,7 @@ export default class TimeGrid extends DateComponent<TimeGridProps> {
     }
 
     this.nowIndicatorEls = nodes
+    this.publiclyTrigger('nowIndicatorRender', [{ view: this.view }])
```
Comment (arshaw) [inline packages/timegrid/src/TimeGrid.ts]: can you pass in the `nodes` to the callback? call the property `els`?

**Comment by zaalbarxx** [comment]
@arshaw should be good now. Not sure if I should also create pull request for updating TS typings ?

**Comment by arshaw** [comment]
these render hooks are now available in v5:
https://fullcalendar.io/docs/now-indicator-render-hooks

thanks anyway for the PR @zaalbarxx !

**Pull request closed without merging**